docs/usage.md

```diff
@@ -164,6 +164,160 @@ $person = $this->converter->convert($user);
 
 Conversion done.
 
+## Special Populators
```

Comment:
Here are the docs from the other PR included, right? So we have to merge this first and then rebase this one here correctly.